lib/hero-image-header/index.js

```diff
@@ -21,7 +21,8 @@ export default class Header extends Component {
   }
   render () {
     const wrapperStyle = {
-      backgroundImage: `url(${heroImageUrl})`
+      backgroundImage: `url(${heroImageUrl[0]})`,
```

Comment:
> @jackcarlisle's point regarding the hero image being _hard-coded_ ... 
My _understanding_ of this code is that the `0` index is the url and `1` is the `background-color` ...